purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -107,6 +108,8 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             afterSetListener(value)
         }
 
+    private val lifecycleHandler = AppLifecycleHandler(this)
```

Comment:
`AppLifecycleHandler` should be `AppLifeCycleHandler` to be consistent with your naming of other `LifeCycle` classes.

Reply:
I am actually going to rename LifeCycle to Lifecycle, to match Android's way